### PR TITLE
Add unit test for ipoib package

### PR DESCRIPTION
**Pull request by Mmduh-483**

### CODE REVIEW

File: pkg/ipoib/ipoib.go [hunk 1]
```diff
@@ -47,38 +93,41 @@ func (im *ipoibManager) CreateIpoibLink(conf *types.NetConf, ifName string, netn
...
+		if err := im.nLink.LinkSetUp(link); err != nil {
+			return err
+		}
 		iface.Name = ifName
 
-		ipoibContLink, err := netlink.LinkByName(ifName)
 		if err != nil {
 			return fmt.Errorf("failed to refetch interface %q: %v", ifName, err)
 		}
-		iface.Mac = ipoibContLink.Attrs().HardwareAddr.String()
```
Comment (moshe010) [inline pkg/ipoib/ipoib.go]: why you remove this?

File: pkg/ipoib/ipoib.go [hunk 2]
```diff
@@ -47,38 +93,41 @@ func (im *ipoibManager) CreateIpoibLink(conf *types.NetConf, ifName string, netn
...
+			_ = im.nLink.LinkDel(ipoibLink)
 			return fmt.Errorf("failed to rename interface to %q: %v", ifName, err)
 		}
+		if err := im.nLink.LinkSetUp(link); err != nil {
+			return err
+		}
 		iface.Name = ifName
 
-		ipoibContLink, err := netlink.LinkByName(ifName)
 		if err != nil {
```
Comment (moshe010) [inline pkg/ipoib/ipoib.go]: check error of what? you remove the "ipoibContLink, err := netlink.LinkByName(ifName)"

File: pkg/ipoib/ipoib.go (hunk 1 above)
Comment (Mmduh-483) [reply pkg/ipoib/ipoib.go]: by mistake

File: pkg/ipoib/ipoib.go (hunk 2 above)
Comment (Mmduh-483) [reply pkg/ipoib/ipoib.go]: by mistake

**Pull request merged**